app/page.tsx

```diff
@@ -141,12 +139,12 @@ export default function IndexPage() {
       </div>
 
       <div className="bg-radial-gradient flex flex-col gap-32 px-6 py-24 md:px-12">
-        <section className="relative grid w-full grid-cols-1 gap-10 overflow-hidden lg:grid-cols-4 lg:gap-0">
-          <h2 className="text-xl uppercase text-orange lg:order-none lg:text-center">
+        <section className="relative grid w-full grid-cols-1 gap-10 overflow-hidden lg:grid-cols-3 lg:gap-0">
+          <h6 className="flex w-full justify-start text-xl uppercase text-orange lg:justify-center">
             Who we are
-          </h2>
-          <div className="lg:col-span-3 lg:pl-[140px]">
-            <h3 className="max-w-2xl text-3xl font-bold">
+          </h6>
+          <div className="col-span-0 flex flex-col lg:col-span-1">
+            <h3 className="text-3xl font-[700] text-black">
```

Comment:
```suggestion
            <h3 className="text-3xl font-bold">
```

don't need to add `text-black` and if you do want to add a color for text use `text-slate-900` instead